src/common/strtol.cc

```diff
@@ -148,41 +148,47 @@ T strict_iec_cast(std::string_view str, std::string *err)
     unit = str.substr(u, str.length() - u);
     // we accept both old si prefixes as well as the proper iec prefixes
     // i.e. K, M, ... and Ki, Mi, ...
-    if (unit.back() == 'i') {
-      if (unit.front() == 'B') {
-        *err = "strict_iecstrtoll: illegal prefix \"Bi\"";
-        return 0;
-      }
-    }
     if (unit.length() > 2) {
```

Comment:
What about `1KiB`? (where `unit.length() == 3`)?

Reply:
handled cases like KB, MB and KiB, MiB etc.